lib/r10k/action/runner.rb

```diff
@@ -30,6 +37,19 @@ def setup_logging
       end
 
       def setup_settings
+        setup_authorization
+      end
+
+      def setup_authorization
+        begin
+          license = R10K::Util::License.load
+
+          if license.respond_to?(:authorization_token)
```

Comment:
Why are we checking to see if a method on `license` was defined rather than seeing if the returned license was/was not nil?